9/changelog.yaml

```diff
@@ -24,6 +24,8 @@ releases:
           Users can still install this collection with ``ansible-galaxy collection install community.skydive``."
         - The deprecated servicenow.servicenow collection has been removed from Ansible 7, but accidentally re-added to Ansible 8. It has been removed again from Ansible 9
           (https://github.com/ansible-community/community-topics/issues/246).
+        - The ngine_io.vultr collection has been removed from Ansible 9 because it is officially unmaintained and has been archived.
```

Comment:
Should this entry also link to https://github.com/ansible-community/community-topics/issues/257?

Reply:
Good point, I've added the link.